dev-libs/libsecp256k1/metadata.xml

```diff
@@ -10,7 +10,6 @@
     <name>Proxy Maintainers</name>
   </maintainer>
   <use>
-    <flag name="asm">Enable x86_64 assembly optimisation</flag>
```

Comment:
**Note:** The local `asm` `USE` flag is dropped because `asm` became a global `USE` flag in 1de89ef8f83a38d45f3f7435d1ec55027be5b394.